src/ImageSharp/ImageInfo.cs

```diff
@@ -3,26 +3,26 @@
 
 using SixLabors.ImageSharp.Formats;
 using SixLabors.ImageSharp.MetaData;
+using SixLabors.Primitives;
 
 namespace SixLabors.ImageSharp
 {
     /// <summary>
     /// Contains information about the image including dimensions, pixel type information and additional metadata
     /// </summary>
-    internal sealed class ImageInfo : IImageInfo
+    public abstract class ImageInfo : IImageInfo
```

Comment:
I'm tempted to remove this class and have each format Info implement the interface. The internal constructors on those derived classes allow us the freedom to alter the classes but this classes would now be fixed and exposed.